Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -433,6 +433,10 @@ - (void)showNotificationHeader {
     [header.enableNotificationsButton addTarget:self action:@selector(enableNotificationsButtonPressed) forControlEvents:UIControlEventTouchUpInside];
 
     [[NSUserDefaults wmf_userDefaults] wmf_setDidShowNewsNotificationCardInFeed:YES];
+    
+     if (self.collectionView.contentOffset.y <= 0) {
```

Comment:
minor, but this non-obvious (to me) condition is in 2 places. Does it make sense to refactor into a method?